cmd/app/start.go

```diff
@@ -17,12 +17,26 @@ limitations under the License.
 package app
 
 import (
+	"crypto"
+	"crypto/rand"
+	"crypto/x509"
+	"encoding/pem"
 	"flag"
-
+	"fmt"
+	"github.com/cert-manager/csi-lib/driver"
+	"github.com/cert-manager/csi-lib/manager"
+	"github.com/cert-manager/csi-lib/metadata"
+	"github.com/cert-manager/csi-lib/storage"
+	"github.com/jetstack/cert-manager-csi/pkg/filestore"
+	"github.com/jetstack/cert-manager-csi/pkg/keygen"
+	"github.com/jetstack/cert-manager-csi/pkg/requestgen"
+	cmclient "github.com/jetstack/cert-manager/pkg/client/clientset/versioned"
 	"github.com/spf13/cobra"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/klogr"
+	"k8s.io/utils/clock"
 
 	"github.com/jetstack/cert-manager-csi/cmd/app/options"
-	"github.com/jetstack/cert-manager-csi/pkg/driver"
 )
```

Comment:
I agree, though the intention of this PR is to just get James' PR to pass CI changing to csi-lib. I have cleaned everything up in a later PR. Happy to squash some PRs together if you like.

Reply:
Would've been really useful to have that info in the comment for the original PR so I didn't waste my time making comments like this, then. Without any further instructions, all I can do is take this PR as it comes.

In any case, I'm not going to review this further as-is because that would basically require me to manually check every comment I have against like 10 other PRs. If it's a case of "rubber stamp this and it'll be fixed later" then I'm fine with that, but at the moment there's basically no further info to guide me.

Reply:
Thanks for the feedback, I'll try and make sure all PRs have better comments in future. This PR was picked up by me as a continuation of munnerz'. The intention was the original reviewer was going to be munnerz so less context was needed for the change, but that is no longer the case.

Hopefully the later PRs are more clear.